api/agent/drivers/docker/cookie.go

```diff
@@ -95,9 +95,9 @@ func (c *cookie) configureTmpFs(log logrus.FieldLogger) {
 	var tmpFsOption string
 	if c.task.TmpFsSize() != 0 {
 		if c.drv.conf.MaxTmpFsInodes != 0 {
-			tmpFsOption = fmt.Sprintf("size=%dm,nr_inodes=%d", c.task.TmpFsSize(), c.drv.conf.MaxTmpFsInodes)
+			tmpFsOption = fmt.Sprintf("size=%dk,nr_inodes=%d", c.task.TmpFsSize(), c.drv.conf.MaxTmpFsInodes)
```

Comment:
@skinowski can you check this one? it seems like the test units were wrong and it's weird this worked before this patch (couldn't figure it out, also brain hurts from PR, just need a glance at this units thing here if you don't mind, don't need to review full PR)

Reply:
can, but fwiw a long time ago in a galaxy far far away the idea was to hang all call config off of calls so that for eg async machine configs don't fuck up stuff like this (ie the lb could set them) when they pull a task and are configured differently. $.02